kitsune/users/tests/__init__.py

```diff
@@ -104,3 +105,30 @@ class Meta:
     jwt_id = "e19ed6c5-4816-4171-aa43-56ffe80dbda1"
     issued_at = "1565720808"
     profile = factory.SubFactory(ProfileFactory)
+
+
+class ConversationFactory:
+    """
+    Populate inboxes and outboxes with messages between a primary user and a number of other users.
+    """
+
+    def __init__(self, primary_user=None, number_of_other_users=2):
+        # Set or create the primary user and create some other users.
+        self.user = primary_user or UserFactory()
+        self.other_users = UserFactory.create_batch(number_of_other_users)
+        # Populate the inboxes and outboxes of the users.
+        for sender in self.other_users:
+            send_message([self.user], "foo", sender=sender)
+        send_message(self.other_users, "bar", sender=self.user)
+        # Confirm our expectations.
+        assert self.user.outbox.count() == 1
+        assert self.user.inbox.count() == len(self.other_users)
+        assert self.inboxes_and_outboxes_of_others_are_populated()
+
+    def inbox_and_outbox_of_primary_user_are_empty(self):
```

Comment:
I feel that we should test this in any test that we need to and not in the factory. It's not DRY but for the tests shouldn't be a problem. Let's keep each test as atomic as possible